src/storage/mvcc/txn.rs

```diff
@@ -181,7 +186,11 @@ impl<'a> MvccTxn<'a> {
                               key,
                               self.start_ts,
                               commit_ts);
-                        Err(Error::TxnLockNotFound)
+                        Err(Error::TxnLockNotFound {
+                            start_ts: self.start_ts,
+                            commit_ts: commit_ts,
+                            key: key.encoded().to_owned(),
```

Comment:
The _key_ is passed by reference to this function. And I don't think the error will happen too often so maybe it's acceptable.